game.js

```diff
@@ -11,7 +11,7 @@ var answers = ["Yess, correct he is the one that crafted me", "Yess, correct he
 var els = [ans1, ans2, ans3, bonus]
 
 function game(questions, answers, index) {
-	var ques = prompt(questions).toLowerCase();
+	// var ques = prompt(questions).toLowerCase();
```

Comment:
Looks like you want to fix the settings for tab-space indentation. You're set up for 4-spaces, and it should be two.